### ISSUE TITLE
nightingale不兼容open-falcon的mymon插件？

### ISSUE THREAD

**Issue by crazylsq** [didi/nightingale#27]
测试nightingale 是否可以使用mymon插件，发现数据无法发送到本机的agent

2020/03/25 18:21:02.079 [E] [main.go:150] Send response 127.0.0.1:3306 -
2020/03/25 18:21:02.673 [E] [metric.go:333] Send alive data failed: Post http://127.0.0.1:2058/api/collector/push

是否是插件不兼容？

**Comment by UlricQin** [comment]
@heli567 这两天会着重帮忙看看mymon的问题，稍等 @crazylsq

**Comment by crazylsq** [comment]
感谢

**Comment by UlricQin** [comment]
> 感谢

把更详细的日志贴一下也？

**Comment by UlricQin** [comment]
> 测试nightingale 是否可以使用mymon插件，发现数据无法发送到本机的agent
> 
> 2020/03/25 18:21:02.079 [E] [main.go:150] Send response 127.0.0.1:3306 -
> 2020/03/25 18:21:02.673 [E] [metric.go:333] Send alive data failed: Post http://127.0.0.1:2058/api/collector/push
> 
> 是否是插件不兼容？

把更详细的日志贴一下也？

**Comment by crazylsq** [comment]
2020/03/26 11:25:22.295 [D] [senddata.go:46] send data to falcon-agent error: Post http://127.0.0.1:2058/api/collector/push: read tcp 127.0.0.1:33442->127.0.0.1:2058: read: connection reset by peer
2020/03/26 11:25:22.295 [E] [main.go:150] Send response 127.0.0.1:3306 -
2020/03/26 11:25:22.304 [I] [senddata.go:155] Error remove ./snapshot/process_127.0.0.1:3306_16, remove ./snapshot/process_127.0.0.1:3306_16: no such file or directory
2020/03/26 11:25:22.304 [D] [senddata.go:82] ignorefile ./falconignore does not exist
2020/03/26 11:25:22.304 [I] [senddata.go:39] Send to http://127.0.0.1:2058/api/collector/push, size: 1
2020/03/26 11:25:22.304 [I] [senddata.go:41] &{mysql_alive_local test-mysql-01 1 GAUGE port=3306,isSlave=0,readOnly=0,type=mysql 1585193122 60}
2020/03/26 11:25:22.306 [D] [senddata.go:46] send data to falcon-agent error: Post http://127.0.0.1:2058/api/collector/push: EOF
2020/03/26 11:25:22.306 [E] [metric.go:333] Send alive data failed: Post http://127.0.0.1:2058/api/collector/push: EOF

**Comment by UlricQin** [comment]
是mymon的问题，post数据的时候传的header不对， @heli567 已经修改，放到了n9e/mymon下
